package.json

```diff
@@ -38,6 +38,9 @@
     "@babel/plugin-transform-runtime": "^7.9.0",
     "@babel/preset-env": "^7.9.0",
     "@babel/runtime": "^7.9.2",
+    "@vue/cli-plugin-unit-jest": "^4.3.1",
```

Comment:
missed one

Reply:
Test script error-ed if i remove this dependency.

Reply:
vue-jest < 4.0 requires the older `babel-core` (not `@babel/core`) https://github.com/vuejs/vue-jest/ issues/160, try installing `babel-bridge`

Reply:
Works. 😄  For my own learning, how did you figure this out? You checked the issues in vue-jest if someone faced the same problem as me?

Reply:
just googled from a hunch and found that link. I previously upgrade `babel-core` to `@babel/core` in the webpack pr so its still fresh 😏